autoload/pydocstring.vim

```diff
@@ -121,10 +122,8 @@ function! s:builddocstring(strs, indent)
           let arg = substitute(line, '{{_args_}}', arg, '')
           call add(docstrings, a:indent . arg)
         endfor
-      elseif line == '"""'
-        call add(docstrings, a:indent . line)
       else
-        call add(docstrings, line)
+        call add(docstrings, a:indent . line)
```

Comment:
Here we indent everything else in the docstring template as well, because it should be indented.